plugins/modules/consul_session.py

```diff
@@ -147,14 +148,14 @@
     ttl: 600  # sec
 '''
 
+from ansible.module_utils.basic import AnsibleModule
+
 try:
-    import consul
+    import requests
```

Comment:
Do I understand correctly that `requests` is a dependency of `consul`? If no, this would be a breaking change.

Reply:
Correct, requests is already an existing dependency of consul.

Since python-consul is not longer used by consul_session, I updated the requirements section of the documentation to reflect that.

Reply:
Thanks for confirming this!